firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageViewController.swift

```diff
@@ -28,20 +28,28 @@ final class NativeErrorPageViewController: UIViewController,
     // MARK: UI Elements
     private struct UX {
         static let logoSizeWidth: CGFloat = 221
+        static let logoSizeWidthiPad: CGFloat = 240
```

Comment:
Minor nitpick: I think our typical naming scheme usually capitalizes iPad in camel case variables (`logoSizeWidthIPad`). NAB.